projects/plugins/boost/tests/e2e/specs/jetpack-compatibility.test.js

```diff
@@ -0,0 +1,57 @@
+import { test, expect } from '../fixtures/base-test.js';
+import { JetpackBoostPage } from '../lib/pages/index.js';
+import { prerequisitesBuilder, isModuleActive } from 'jetpack-e2e-commons/env/prerequisites.js';
+import { activateModules, deactivateModules } from 'jetpack-e2e-commons/env/index.js';
+import {
+	boostPrerequisitesBuilder,
+	activateModules as activateBoostModules,
+	deactivateModules as deactivateBoostModules,
+} from '../lib/env/prerequisites.js';
+
+let jetpackBoostPage;
```

Comment:
Is there a reason for this variable to be global?

Reply:
Started with this to manipulate the same page with multiple tests. It is no more valid. Thanks for pointing out @brbrr